### PR TITLE
Embed the git commit id in a go file and print it on startup.

### LINKED ISSUE #88: Embed changeset id and log it on startup

### PR DESCRIPTION
**Pull request by tomwilkie**
Fixes #88

### CODE REVIEW

**Comment by peterbourgon** [comment]
Oof. That's awkward. Let's have a [version variable](https://github.com/soundcloud/glimpse/blob/master/agent/main.go#L28) that's [set at build-time](https://github.com/soundcloud/glimpse/blob/master/agent/Makefile#L20).

**Comment by tomwilkie** [comment]
Yeah thats much cleaner.  PTAL

File: app/main.go [hunk 1]
```diff
@@ -63,7 +66,7 @@ func main() {
 		defer os.Remove(*pidfile)
 	}
 
-	log.Printf("starting")
+	log.Printf("app starting, commit id %s", version)
```
Comment (peterbourgon) [inline app/main.go]:
```go
log.Printf("starting, version %s", version)
```

File: probe/main.go [hunk 2]
```diff
@@ -39,6 +42,8 @@ func main() {
 		os.Exit(1)
 	}
 
+	log.Printf("probe starting, commit id %s", version)
```
Comment (peterbourgon) [inline probe/main.go]:
```go
log.Printf("starting, version %s", version)
```

**Comment by peterbourgon** [comment]
Minor formatting nit aside, :dango:

File: app/main.go (hunk 1 above)
Comment (tomwilkie) [inline app/main.go]: I put app and probe in their as their logs get mixed up when run under runit

**Pull request merged**